### ISSUE TITLE
Setting flag to welsh translates partially main service 

### ISSUE THREAD

**Issue by Marat-Faizov** [hmrc/accessibility#61]
### Problem 
Users able to set language to welsh on the cookie page and accessibility statement page. After moving to the main service the header/footer still persist the Welsh whilst the page body shows English and lang=en not applied

### Solution
One of the possible solutions to pass separate lang=en to the `main` element, but other elements outside of `main` will still violate WCAG 3.1.2 Language of parts such as breadcrumbs service name etc

Jira ticket created by plat-ui team https://jira.tools.tax.service.gov.uk/browse/PLATUI-1163

![image](https://user-images.githubusercontent.com/77624397/117457047-c4d51a00-af40-11eb-8136-5c33c0ab40c4.png)

**Comment by Marat-Faizov** [comment]
setting play.i18n.langs = ["en"] with `welshToggleEnabvled=false` in application.conf fixes the issue.

**Comment by ashfaqhussain357** [comment]
Jira ticket closed for this issue by Robert Best with the following comments -  Closing this ticket as no apparent requirement for providing partially translated web pages.
